psydac/linalg/tests/test_stencil_matrix.py

```diff
@@ -433,7 +433,7 @@ def test_stencil_matrix_2d_serial_spurious_entries( dtype, p1, p2, s1, s2, P1, P
 @pytest.mark.parametrize('dtype', [float])
 @pytest.mark.parametrize('n1', [7])
 @pytest.mark.parametrize('p1', [1,2])
-@pytest.mark.parametrize('s1', [1])
+@pytest.mark.parametrize('s1', [1,2])
```

Comment:
Minor suggestion on style:
```suggestion
@pytest.mark.parametrize('s1', [1, 2])
```